### ISSUE TITLE
Main website accessibility issue WCAG 1.4.3

### ISSUE THREAD

**Issue by A11Ybot-OP** [knative/docs#4811]
Path
.md-header__source > .md-source[title="Go\ to\ repository"][data-md-component="source"] > .md-source__repository[data-md-state="done"] > .md-source__facts > .md-source__fact--stars.md-source__fact

Snippet
<li class="md-source__fact md-source__fact--stars">3.3k</li>

How to fix
Fix the following:
Element has insufficient color contrast of 4.31 (foreground color: #3f75ac, background color: #f3f1f2, font size: 8.3pt (11px), font weight: bold). Expected contrast ratio of 4.5:1Element has insufficient color contrast of 4.31 (foreground color: #3f75ac, background color: #f3f1f2, font size: 8.3pt (11px), font weight: bold). Expected contrast ratio of 4.5:1
Use foreground color: #3b6fa3 and the original background color: #f3f1f2 to meet a contrast ratio of 4.68:1Use foreground color: #3b6fa3 and the original background color: #f3f1f2 to meet a contrast ratio of 4.68:1
Or use background color: #fffdfe and the original foreground color: #3f75ac to meet a contrast ratio of 4.76:1.

**Comment by A11Ybot-OP** [comment]
there are 17 other similar colour contrast issues

**Comment by abrennan89** [comment]
@csantanapr I think this one should be moved to UX instead of docs?

**Comment by abrennan89** [comment]
Issue moved to [knative/ux #68](https://github.com/knative/ux/issues/68) via [**ZenHub**](https://www.zenhub.com/)

**Comment by csantanapr** [comment]
This problem is something that affects the website and any changes to fix this will happened with a PR and code changes on the docs repo 

So the discussion around the website content, usability, and infra of it should remain on this repo
